### DISCUSSION TITLE
TTF rendering problem.

### DISCUSSION THREAD

**Discussion by night2876** [xoreos/xoreos#589] [General]
![nwn2](https://user-images.githubusercontent.com/103774955/231531030-d3861115-566e-4862-bffd-fa7d92087d79.jpg)

Hi,

continuing with the GUI but when using ttf fonts with freetype library the rendering have many artefact like in the screen. Someouse could help me with this ?

Thanks in advance.

**Comment by DrMcCoy** [comment]
Hmm, weird, never seen that before. Do you the same artifacts with other ttf fonts? For example, if you pull open the debug console (ctrl+d)?

**Reply by night2876** [reply]
Nope with the console is all ok.
